lib/checkPullRequestLabels.js

```diff
@@ -33,6 +33,7 @@ const {
 } = require('../userWhitelist.json');
 const DEFAULT_CHANGELOG_LABEL = 'REVIEWERS: Please add changelog label';
 const PR_LABELS = ['dependencies', 'stale', DEFAULT_CHANGELOG_LABEL];
+const MILESTONES = ['Blocking Bugs'];
```

Comment:
Can we get the milestones from GitHub? If not this should be probably added as a constant in _constants.js_.